components/logins/src/ffi.rs

```diff
@@ -28,9 +28,8 @@ pub mod error_codes {
     /// Attempted to insert or update a record so that it is invalid
     pub const INVALID_LOGIN: i32 = 4;
 
-    /// Either the file is not a database, or it is not encrypted with the
-    /// provided encryption key.
-    pub const INVALID_KEY: i32 = 5;
+    /// The file is not a database
```

Comment:
Logins can still fail if an incorrect key is specified, right? We should probably keep that in the error message here.

Reply:
Ah, whoops, didn't notice which file I was in 😓